core/src/main/java/software/amazon/awssdk/core/auth/ContainerCredentialsProvider.java

```diff
@@ -77,7 +74,7 @@ protected CredentialsEndpointProvider getCredentialsEndpointProvider() {
 
     @Override
     public String toString() {
-        return "ContainerCredentialsProvider(" + credentialsEndpointProvider.toString() + ")";
+        return "ContainerCredentialsProvider()";
```

Comment:
No container information?

Reply:
It seemed weird. The API to change it is only for testing, so it would always be the same thing: 
`ContainerCredentialsProvider(ContainerCredentialsEndpointProvider)`

The other credential providers were showing content in their parenthesis (eg. the static credentials provider shows the credentials it contains), and this one was showing an implementation detail. I figured it would be more consistent to exclude it. 

Maybe instead we could show the actual endpoint we check? `ContainerCredentialsProvider(http://169.254.170.2/whatever)`